src/template.js

```diff
@@ -339,10 +341,12 @@ module.exports = {
         'p-rep-email-address': repEmailAddress.section,
         'p-rep-telephone-number': repTelephoneNumber.section,
         'p-rep-reference-number': repReferenceNumber.section,
-        'p-rep-claims-management-reg': repClaimsManagementRegNumber.section
+        'p-rep-claims-management-reg': repClaimsManagementRegNumber.section,
+        'p--new-or-existing-application': newOrExistingApp.section,
+        'p--contact-cica': contactCica.section
     },
     routes: {
-        initial: 'p-applicant-fatal-claim',
+        initial: 'p--new-or-existing-application',
```

Comment:
change this back to p-applicant-fatal-claim

Reply:
This is the new first page of the application, so it should replace `p-applicant-fatal-claim` as the initial page.